src/api/schema/events/log.rs

```diff
@@ -71,7 +68,10 @@ impl Log {
     }
 
     /// Get JSON field data on the log event, by field name
-    async fn json(&self, field: String) -> Option<&Value> {
-        self.event.get(field.as_str())
+    async fn json(&self, field: String) -> Option<String> {
+        self.event.get(field.as_str()).map(|field| {
+            serde_json::to_string(field)
```

Comment:
I'm not super familiar with this area of the code but, could this potentially be a performance hit since we are now allocating ?

Reply:
Good question. I am also not super familiar with `async-graphql` but seemingly it would have had to allocate this same string anyway as previously we were passing a `&Value` and then it converted that to a JSON string. Here I'm just doing that up-front and giving it an owned string back.